utilsCSS/utilityClasses.css

```diff
@@ -114,3 +114,22 @@
 .spacing-xxl {
   padding: 4rem !important;
 }
+
+.font-xs {
+  font-size: 0.5rem !important;
```

Comment:
"Best practices dictate that ' **!important** 'should be used sparingly, if at all, and certainly never just use to get yourself out of a sticky situation with inheritance and the cascade."  Quoting this from a book.

Use of '!importance' is very rare. If we can avoid using this, it is one of the best practices. 

I suggest to go over this once more and get back to me. I will approve after that.

Reply:
Yes important should be avoided

Reply:
In general, I agree with you, When we say not use important we are taking about nested classes, But in this particular use case, The thought process is to override the font sizes defined in the classes of elements.

Here's a blog talking about using ! Important in utility classes https://www.cssreactions.com/views/important.html